resources/hooks/local/pre-commit

```diff
@@ -12,4 +12,4 @@ DIFF=$(git -c diff.mnemonicprefix=false --no-pager diff -r -p -m -M --full-index
 export GRUMPHP_GIT_WORKING_DIR=$(git rev-parse --show-toplevel)
 
 # Run GrumPHP
-(cd "${HOOK_EXEC_PATH}" && printf "%s\n" "${DIFF}" | $(EXEC_GRUMPHP_COMMAND) $(HOOK_COMMAND) '--skip-success-output')
+(cd "${HOOK_EXEC_PATH}" && printf "%s\n" "${DIFF}" | $(EXEC_GRUMPHP_COMMAND) $(HOOK_COMMAND) '--skip-success-output' '--no-interaction')
```

Comment:
It is better to move this additional parameter (and possibly the `--skip-success-output` as well to the hook builder:

https://github.com/phpro/grumphp/blob/2ab78e5bdc08d4a579d8e7c12575a3203c88a95e/src/Console/Command/Git/InitCommand.php#L159-L175

This way, all hooks contain the same parameters.